source/guides/getting-started.md

```diff
@@ -713,15 +713,16 @@ require 'spec_helper'
 require_relative '../../../../apps/web/controllers/books/create'
 
 describe Web::Controllers::Books::Create do
+  let(:action) { Web::Controllers::Books::Create.new }
   let(:params) { Hash[book: { title: 'Confident Ruby', author: 'Avdi Grimm' }] }
 
-  after do
+  before do
     BookRepository.clear
   end
 
   it 'creates a new book' do
     action.call(params)
-    action.book.id.wont_be_nil
+    BookRepository.all.count.wont_equal 0
```

Comment:
I don't think there is a "lotus" way for doing this, maybe it's better to extract the logic to BookCreation interactor class and test that controller has called that class once.